### PR TITLE
Enable live poll from url params

### PR DESCRIPTION
**Pull request by Benratelade**
This is related to this discussion: 
https://github.com/mperham/sidekiq/discussions/5197#discussioncomment-2491400

When viewing the queued jobs in the web view, one can pass a `?poll=true` parameter in the URL which will enable polling.

### CODE REVIEW

File: myapp/db/schema.rb [hunk 1]
```diff
@@ -10,13 +10,12 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2012_01_23_214055) do
```
Comment (mperham) [inline myapp/db/schema.rb:13]: This shouldn't be part of this PR.

**Comment by Benratelade** [comment]
noted @mperham, I undid those changes to schema.rb.
